narwhal/node/src/lib.rs

```diff
@@ -218,16 +218,16 @@ impl Node {
 
         // Spawn the consensus core who only sequences transactions.
         let ordering_engine = Bullshark::new(
-            (**committee.load()).clone(),
+            (*committee).clone(),
             store.consensus_store.clone(),
             parameters.gc_depth,
             consensus_metrics.clone(),
         );
         let consensus_handles = Consensus::spawn(
-            (**committee.load()).clone(),
+            (*committee).clone(),
             store.consensus_store.clone(),
             store.certificate_store.clone(),
-            tx_reconfigure.subscribe(),
+            tx_shutdown.subscribe(),
```

Comment:
nit: let's also unify our approach to either pass receivers, or senders which are then used to subscribe in the spawned component (this is the only outlier?)